tests/openstack/task.yaml

```diff
@@ -0,0 +1,101 @@
+summary: Test the openstack backend.
+
+systems: [ubuntu-22.04-64-devstack]
+
+kill-timeout: 30m
+
+prepare: |
+    if [ ! -f .spread-reuse.yaml ]; then
+        touch /run/spread-reuse.yaml
+        ln -s /run/spread-reuse.yaml .spread-reuse.yaml
+    fi
+
+    sudo -u stack -i /opt/stack/devstack/unstack.sh
+    sudo -u stack -i /opt/stack/devstack/stack.sh
+
+execute: |
+    # Check the instances which cannot be accessed are discarded
+    # The instance was created and the status has to be active to
+    # fail trying to access through ssh
+    spread openstack:cirros-64: -v -reuse -resend &> task.out || true
+    grep 'cannot find ready marker in console output for .*: timeout reached' task.out
+
+    # Check that the server is discarded during the spread execution
+    for i in $(seq 10); do
+        if [ -z "$(openstack server list)" ]; then
+            break
+        fi
+        sleep 1
+    done
+    test -z "$(openstack server list)"
```

Comment:
Is there a prior check that ensures that the list started empty in the first place?  Also, please consider the comment above in this context.

Also, won't OpenStack show the any used servers as terminated instead of just showing an empty list?

Reply:
Added the initial check to verify there is not any server running. 

The command `openstack server list` doesn't show terminated servers, it just includes active ones.